x-pack/test/functional/apps/ml/anomaly_detection/single_metric_viewer.ts

```diff
@@ -39,51 +39,168 @@ export default function ({ getService }: FtrProviderContext) {
+          time_field: 'order_date',
+          time_format: 'epoch_ms',
+        },
+        analysis_limits: {
+          model_memory_limit: '11mb',
+          categorization_examples_limit: 4,
+        },
+        model_plot_config: { enabled: true },
+      };
+
+      const datafeedConfig: Datafeed = {
+        datafeed_id: 'datafeed-ecom_01',
+        indices: ['ft_ecommerce'],
+        job_id: 'ecom_01',
+        query: { bool: { must: [{ match_all: {} }] } },
+      };
+
+      before(async () => {
+        await esArchiver.loadIfNeeded('ml/ecommerce');
+        await ml.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+        await ml.testResources.setKibanaTimeZoneToUTC();
+        await ml.api.createAndRunAnomalyDetectionLookbackJob(jobConfig, datafeedConfig);
+        await ml.securityUI.loginAsMlPowerUser();
+      });
+
+      after(async () => {
+        await ml.api.cleanMlIndices();
+      });
+
+      it('opens a job from job list link', async () => {
+        await ml.testExecution.logTestStep('navigate to job list');
+        await ml.navigation.navigateToMl();
+        await ml.navigation.navigateToJobManagement();
+
+        await ml.testExecution.logTestStep('open job in single metric viewer');
+        await ml.jobTable.waitForJobsToLoad();
+        await ml.jobTable.filterWithSearchString(jobConfig.job_id, 1);
+
+        await ml.jobTable.clickOpenJobInSingleMetricViewerButton(jobConfig.job_id);
+        await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+      });
+
+      it('render entity control', async () => {
+        await ml.testExecution.logTestStep('pre-fills the detector input');
+        await ml.singleMetricViewer.assertDetectorInputExist();
+        await ml.singleMetricViewer.assertDetectorInputValue('0');
+
+        await ml.testExecution.logTestStep('should input entities values');
+        await ml.singleMetricViewer.assertEntityInputExist('day_of_week');
+        await ml.singleMetricViewer.assertEntityInputSelection('day_of_week', []);
+        await ml.singleMetricViewer.selectEntityValue('day_of_week', 'Friday');
+        await ml.singleMetricViewer.assertEntityInputExist('geoip.city_name');
+        await ml.singleMetricViewer.assertEntityInputSelection('geoip.city_name', []);
+        await ml.singleMetricViewer.selectEntityValue('geoip.city_name', 'Abu Dhabi');
+
+        // TODO if placed before combobox update, tests fail to update combobox values
+        await ml.testExecution.logTestStep('assert the default state of entity configs');
+        await ml.singleMetricViewer.assertEntityConfig(
+          'day_of_week',
+          true,
+          'anomaly_score',
+          'desc'
+        );
+
+        await ml.singleMetricViewer.assertEntityConfig(
+          'geoip.city_name',
+          true,
+          'anomaly_score',
+          'desc'
+        );
+
+        await ml.singleMetricViewer.setEntityConfig('geoip.city_name', false, 'name', 'asc');
```

Comment:
Would be good to have another `logTestStep` here with something like `modify the entity config`.

Reply:
Added in https://github.com/elastic/kibana/pull/81510/commits/ca7c2fc0382dd05b98395d6264db8f2a43240f21